components/dashboard/package.json

```diff
@@ -9,13 +9,15 @@
     "@gitpod/public-api": "0.1.5",
     "@stripe/react-stripe-js": "^1.7.2",
     "@stripe/stripe-js": "^1.29.0",
+    "@types/react-datepicker": "^4.8.0",
```

Comment:
**suggestion**: I assume the library we chose helps us bootstrap this, but could we use a better alternative, like the **[date picker component](https://github.com/duetds/date-picker)** from the duet design system, which is also open source? Alternatively, we could go with our current option and try to improve the style and accessibility issues around it.

Reply:
I tried that lib a little but struggled to make it work with react and typescript.

Reply:
Thanks for giving this a try! 🔥 😭 

I'd love us to move away from the existing one, but like you [mentioned above](https://github.com/gitpod-io/gitpod/pull/14515#discussion_r1019110025), let's wait until it's worth investing time and effort to design one for our needs. 🤝 

In the meantime, from https://github.com/gitpod-io/gitpod/pull/14515#discussion_r1019119043:
> Let's try to bring the current library to offer a bare minimum good UX. I can try to make some visual changes in a follow-up PR if needed.